src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,65 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { useDispatch } from 'react-redux';
+import { useAppSelector } from '../../app/hooks';
+import { setCurrentTodo } from '../../features/currentTodo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
 
 export const TodoModal: React.FC = () => {
+  const dispatch = useDispatch();
+  const [isLoading, setIsLoading] = useState(false);
+  const [currentUser, setCurrentUser] = useState<User | null>(null);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getUser(currentTodo?.userId || 0)
+      .then(setCurrentUser)
+      .finally(() => setIsLoading(false));
```

Comment:
The `getUser` function call does not handle errors. Consider adding a `catch` block to handle any potential errors that may occur during the API call, such as displaying an error message or setting an error state.